client/verta/verta/client.py

```diff
@@ -3360,7 +3360,81 @@ def get_artifact(self, key):
             except:
                 return six.BytesIO(artifact)
 
-    # TODO: download_artifact(self, key, download_to_path)
+    def download_artifact(self, key, download_to_path, chunk_size=32*(10**3)):
```

Comment:
```suggestion
    def download_artifact(self, key, download_to_path, chunk_size=32*(10**6)):
```
KB will cause way too much IO